README.md

```diff
@@ -267,15 +281,15 @@ You can also use random test data by specifying the groupCode or riderId to be "
 Random group codes are generated by combing a groupCodeBase ("BIG" in the above example) with a random number. The above JSON would result in a random selection between BIG1, BIG2, BIG3, ..., BIG50.
 
 Random rider IDs are selected from the specified range of values.
- 
+
 ### Recording and Analyzing
 
 To look at the Server performance in real time the linux command `top` is used. When running
 the load tests it is important to record the performance of the server. When you run the
 load test make sure you run top on the server simultaneously in order to record the
 performance of the server at the time of the tests. By using this command and
-pipping the results this is achievable to analyze the data. Then after recording, 
-grep the text in order to get the parameters you want then simply graph the numbers. 
+pipping the results this is achievable to analyze the data. Then after recording,
```

Comment:
I didn't actually type this part. It must be a line ending thing. I have not gone through the analysis dashboard stuff. Should that be included in the ticket?